### PR TITLE
Candidate for the v0.10.12 release tag

### PR DESCRIPTION
**Pull request by kbenzie**
* #1951

### CODE REVIEW

**Review by callumfare — APPROVED** [review]

**Comment by kbenzie** [comment]
Making this draft until we get a green light.

**Comment by nrspruit** [comment]
> Making this draft until we get a green light.

Markes ready to merge after being given the permission to move ahead with this change.

**Pull request merged**
